ParquetClassLibrary/Crafting/CraftingRecipe.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections.Generic;
+using ParquetClassLibrary.Sandbox.ID;
+
+namespace ParquetClassLibrary.Crafting
+{
+    /// <summary>
+    /// Models the ingredients and process needed to produce a new item.
+    /// </summary>
+    public struct CraftingRecipe
+    {
+        #region Defaults
+        /// <summary>Maximum width of the <see cref="StrikePanel"/> patten.</summary>
+        public const int PanelPatternMaxWidth = 2;
+
+        /// <summary>Maximum height of the <see cref="StrikePanel"/> patten.</summary>
+        public const int PanelPatternMaxHeight = 4;
+        #endregion
+
+        /// <summary>The type of item created by following this recipe.</summary>
+        public EntityID ItemProduced { get; }
```

Comment:
Consider if you want recipe fall-offs. For example, imagine the following recipe;
- In: Bucket of Water, Flour
- Out: Dough

...but what happens to the bucket of water? Perhaps it comes back as a Bucket (without water). If you want to have that modeled, maybe a recipe can have multiple outputs.

This might be more of a *tool* thing, than an additional output thing. Worth considering, though.

Reply:
Oh thanks, yes, this is an important point!

Reply:
Added an issue for this.